src/api/toggleMobilePushSettings.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import { apiPatch } from '../api/apiFetch';
+import type { Auth } from '../types';
+
+export default async (
+  {
+    auth,
+    offline,
+    online,
+  }: { auth: Auth, offline?: boolean, online?: boolean, }) =>
+  apiPatch(
+    auth,
+    'settings/notifications',
+    res => res,
+    JSON.parse(JSON.stringify({ // To remove undefined values
```

Comment:
Extract that to an utility function that is called `removeUndefinedValues` then you don't need a comment.